src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -6943,6 +6972,21 @@ instruct loadConD(vRegD dst, immD con) %{
   ins_pipe(fp_load_constant_d);
 %}
 
+// Load Half Float Constant
+// ldr instruction has 32/64/128 bit variants but not a 16-bit variant. This
+// loads the 16-bit value from constant pool into a 32-bit register but only
+// the bottom half will be populated.
```

Comment:
Surely what actually happens here is that it loads a 32-bit word from the constant pool. The bottom 16 bits of this word contain the half-precision constant.

Reply:
I agree. The wording didn't quite convey that. I will change it in my next PS. Thank you for looking into the patch!

Reply:
Addressed this in the new patch.